pallets/parachain-staking/README.md

```diff
@@ -1,7 +1,131 @@
+     total_staking_reward is set to 0.
+
+4. The Candidate's reward shared
+is further divided proportionally based on their **total stake**, which includes:
+   - **Candidate Stake**: The amount staked by the collator themselves.
+   - **Nominators' Stake**: The total amount staked by nominators backing the collator.
+5. **Candidate Rewards**: Candidate' rewards are proportional to their individual stake relative
+to the total stake backing the collator.
+6. **Nominator Rewards**: Nominators' rewards are proportional to their individual stake
+relative to the total stake backing the collator. This is repeated for each nomination an
+account has on different candidates.
+
+#### Reward Formulas
+total_reward_for_candidate = (collator_points / total_points) * total_staking_reward
+candidate_reward = (candidate_stake / candidate_total_stake) * total_reward_for_candidate
+nomination_reward = (nomination_stake / candidate_total_stake) * total_reward_for_candidate
+
+- `collator_points`: Total points the collator earned in the era.
+- `total_points`: Total points earned by all collators in the era.
+- `total_staking_reward`: Total reward available in the reward pot for the era.
+- `candidate_stake`: The amount staked by the collator themselves.
+- `candidate_total_stake`: Total stake backing the collator (self-bond + nominations stake).
+- `nomination_stake`: Stake contributed by the nominator.
+
+#### Delayed Payouts
+- Rewards are paid after a delay of `T::RewardPaymentDelay` eras.
+- During each block, one collator and their nominators are paid until all rewards for the era
+  are distributed.
+
+### Growth Mechanism
+
+The growth mechanism is an process that increases the total token supply
+across two tiers. A growth period spans multiple eras, defined by `T::ErasPerGrowthPeriod`.
+It rewards collators based on their performance over multiple eras.
+
+#### Conditions
+- **Activation**: The growth mechanism is enabled only if the `T::GrowthEnabled` constant is set
+  to `true`.
+- **Deactivation**: Growth is skipped if:
+   - The number of accumulations is zero.
+   - The total stake or total rewards for the growth period is zero.
+
+#### How It Works
+1. **Triggering Growth**:
+   - If `T::GrowthEnabled` is set to `true`, growth events are triggered every
+     `T::ErasPerGrowthPeriod` eras. These values are configured as part of the pallet setup.
+   - At the beginning of a new era, the system checks whether the conditions for initiating a
+     new growth event are met. If the conditions are satisfied, the `trigger_growth_on_t1`
+     function is invoked. This function sends a request to the bridge contract to mint new
+     tokens. The pallet is not opinionated on how the growth is implemented or how the
+     information provided is used by the formula in tier 1. It simply provides the following
+     data:
+     - `total_stake`: The total average stake in the system at the start of the growth period.
```

Comment:
Is this not the average staked across the period, so calculated at the end of it? 

For example, in a growth period consisting of 3 eras, the following was staked during each era: 

1. 300 staked
2. 400 staked
3. 200 staked

So total staked = 900 / 3 eras = 300 average total staked.

Reply:
yes, that is what I mean with total average stake

Reply:
```suggestion
     - `total_stake`: The mean total stake per era over the growth period.
```